build.zig

```diff
@@ -384,4 +384,8 @@ pub fn build(b: *std.Build) void {
     // Define a single test step that runs all tests
     const test_all_step = b.step("test-all", "Run all unit tests");
     test_all_step.dependOn(test_step);
+
+    const zabi_module = b.dependency("zabi", .{}).module("zabi");
+    exe.root_module.addImport("zabi", zabi_module);
+    lib.root_module.addImport("zabi", zabi_module);
```

Comment:
Btw not sure if you are aware of this but zabi is a modular lib. So for example if you just need the abi utils (which contains the abi json parsing and human-readable parsing)

You can just import it like this:

```zig
exe.root_module.addImport("zabi-abi", zabi.module("zabi-abi"));
lib.root_module.addImport("zabi-abi", zabi.module("zabi-abi"));
```

And you can expand it as you go. If not you can just leave it is and has everything that is available in zabi

Reply:
tyty added zabi in a previous pr